simulation_rate: 250.0 # [Hz]
clock_rate: 250 # [Hz]
realtime_factor: 1.0 # [Hz]

uav_names: [
  "uav1",
]

uav1:
  type: "x500"
  spawn:
    x: 0.0 #10.0
    y: 0.0 #15.0
    z: 2.0
    heading: 0

sensors:
  lidar:
    enabled: true
  rgb:
    enabled: true
  stereo:
    enabled: true
